.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,11 @@
-
-
 ###### Before submitting the PR, please make sure you do the following
+
 - [ ] If you're attempting to fix a translation issue, please submit your changes to our [POEditor project](https://poeditor.com/join/project/ynjQmwSsGh) instead of opening a PR.
+
 ### Tests
--  [ ] Run the frontend and integration tests with  `yarn lint && yarn flow && yarn test`)
--  [ ] If there are changes to the backend codebase, run the backend tests with `lein test && lein lint && ./bin/reflection-linter`
 
--  [ ] Sign the [Contributor License Agreement](https://docs.google.com/a/metabase.com/forms/d/1oV38o7b9ONFSwuzwmERRMi9SYrhYeOrkbmNaq9pOJ_E/viewform)
-(unless it's a tiny documentation change).
+- [ ] Run the frontend and Cypress end-to-end tests with `yarn lint && yarn flow && yarn test`)
```

Comment:
An opportunity to fix this: `yarn lint` already runs Flow (among others, such as ESLint, Prettier, etc).

Reply:
Fixed it. Thank you for valuable suggestion.